media/css/firefox/whatsnew/whatsnew-142.scss

```diff
@@ -1109,12 +1113,6 @@ body {
         text-align: left;
     }
 
-    @media (max-width: 767px) {
```

Comment:
Was removing this intentional?

This made the buttons on the feature elements full width on small screens, which I think is what the Figma design showed, but I may have missed a change request.

Before (with full width buttons):
<img width="402" height="1156" alt="Screenshot 2025-08-21 at 11 33 39 AM" src="https://github.com/user-attachments/assets/ece08337-c217-4ac6-9c0e-649562a4cf12" />

Now (without full width buttons):
<img width="400" height="1156" alt="Screenshot 2025-08-21 at 11 33 28 AM" src="https://github.com/user-attachments/assets/93273d5f-4a4a-43a0-b96e-67a2d6959467" />